src/server.ts

```diff
@@ -86,16 +89,72 @@ app.get("/data/:chainId/rounds/:roundId/applications.csv", async (req, res) => {
       application.metadata.application.project.projectTwitter,
       application.metadata.application.project.projectGithub,
       application.metadata.application.project.userGithub,
-      ...answers,
+      ...answers
     ]);
   }
 
   res.setHeader("content-type", "text/csv");
   res.send(csv.getHeaderString() + csv.stringifyRecords(records));
 });
 
+app.get("/data/:chainId/rounds/:roundId/votecoefficients.csv", async (req, res) => {
```

Comment:
vote and coefficients are two words, we should split them following the convention, like `passport_scores.json`

```suggestion
app.get("/data/:chainId/rounds/:roundId/vote_coefficients.csv", async (req, res) => {
```